services/api/src/utils/mailer.js

```diff
@@ -30,90 +122,70 @@ marked.use({
     link(href, title, text) {
       if (title === '$button$') {
         return `<a href="${href}" class="button" target="_blank"><span class="text">${text}</span></a>`;
+      } else if (href.includes('{{')) {
+        // Links may include template interpolation which
+        // gets escaped so output it unescaped here.
+        return `<a href="${href}">${text}</a>`;
       }
       return false;
     },
   },
 });
 
-const defaultOptions = {
-  appName: config.get('APP_NAME'),
-  appUrl: config.get('APP_URL'),
-  appSupportEmail: config.get('APP_SUPPORT_EMAIL'),
-  appCompanyName: config.get('APP_COMPANY_NAME'),
-  appCompanyAddress: config.get('APP_COMPANY_ADDRESS'),
-  appLogoUrl: config.get('APP_LOGO_URL'),
+// Mustache utils
+
+// Mustache is HTML escaping URLs by default
+// which causes issues so avoid this.
+const escapeHtml = Mustache.escape;
+Mustache.escape = (text) => {
+  if (!text.match(/https?:\/\//)) {
+    text = escapeHtml(text);
+  }
+  return text;
 };
 
+function render(body, options) {
+  if (body) {
+    return Mustache.render(body, options);
+  } else {
+    return '';
```

Comment:
what the case for not having a body?

Reply:
Mostly because I was unsure of the implication of sending `null` or `undefined` to postmark as part of the html or text (might get stringified, you'd get a full html email but the text version would just say "undefined" or something) so I thought I'd normalize to an empty string here as a precaution.

Reply:
make sense